sites/fast-website/src/app/components/content-placement-container/content-placement-container.ts

```diff
@@ -0,0 +1,16 @@
+import { FASTElement, attr, repeat } from "@microsoft/fast-element";
```

Comment:
nitpick, but I do prefer alphabetized input parameters and import statements when possible for better maintainability.

Reply:
I'd expect the build to fail on eslint as a result of this. :)

Reply:
I will go through and make sure that all imports are alphabetized. Thank you for noticing that.

Reply:
@SamanthaAO to save time, I would recommend just running `yarn eslint:fix` in the package root. That should fix any for you and save some time.

Reply:
@chrisdholt I do not believe that that script is available in fast-website.

Reply:
If eslint is included in the package you can run from the CLI:
```
yarn eslint . --ext .ts --fix
```

Reply:
Awesome! thank you!